mmv1/products/cgc/terraform.yaml

```diff
@@ -109,6 +109,40 @@ overrides: !ruby/object:Overrides::ResourceOverrides
           deletion_protection: "false"
         ignore_read_extra:
           - "deletion_protection"
+      - !ruby/object:Provider::Terraform::Examples
+        name: "sql_sqlserver_instance_backup_location"
+        primary_resource_type: "google_sql_database_instance"
+        primary_resource_id: "default"
+        vars:
+          sqlserver_instance_backup_location: "sqlserver-instance-backup-location"
+          deletion_protection: "true"
+        test_vars_overrides:
+          deletion_protection: "false"
+        ignore_read_extra:
+          - "root_password"
+          - "deletion_protection"
+      - !ruby/object:Provider::Terraform::Examples
```

Comment:
Are we able to `skip_test` on the second two? Our tests are pretty expensive to run, and I don't think that these will provide additional signal on provider correctness.

Reply:
Done